build-logic/integration-testing/src/main/kotlin/gradlebuild/integrationtests/extension/IntegrationTestExtension.kt

```diff
@@ -20,6 +20,11 @@ import org.gradle.api.provider.Property
 
 
 abstract class IntegrationTestExtension {
-    abstract val usesJavadocCodeSnippets: Property<Boolean>
+    /**
+     * Enabled by default. If enabled, there will be a `GenerateAutoTestedSamplesTestTask` task
```

Comment:
💭 "Enabled by default" isn't really true, the `integration-tests` build script configures this property to be enabled.  I would expect this doc to mean this property has a convention set in the constructor.